config/routes.rb

```diff
@@ -1,8 +1,11 @@
+# frozen_string_literal: true
+
 Rails.application.routes.draw do
-  devise_for :users, :controllers => {
+  devise_for :users, controllers: {
     registrations: 'registrations'
   }
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  # devise said he wants to have a specified root rout, so:
-  root to: "home#index"
+  root to: 'admins/users#index'
```

Comment:
this is not correct root. Dont push even if it just for easier development. Use git stash